helm/sematic-server/values.yaml

```diff
@@ -1,7 +1,3 @@
-server:
-  sematic_dashboard_url: https://my.sematic # REPLACE ME
```

Comment:
This is required for the Slack integration. Please revert.

Reply:
this was not removed, just moved to the ingress since it's really a property of the external address of the cluster rather than a property of the server deployment itself.  i have explicitly worked to remove this `server:` block in the values yaml since it just becomes a kitchen sink for everything and stops being cleanly organized.